Form-Controls/index.html

```diff
@@ -14,15 +14,80 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
+    <div>
+      <label>Name</label>
```

Comment:
Every input type needs a label so that users can understand what they need to do with the input. To create that relationship between label and input we use the `for=""` attribute.

For example:
`<label for="name">Name</label>`
`<input name="name" id="name" required>`

This is really useful for people who use assistive technology, like blind people who use screen readers and listen the internet because they can't see it.